volunteer_system_user_stories.md

```diff
@@ -3,100 +3,53 @@
 ## User types
 
 * Volunteer - People on the ground helping
-* Trainer - Someone able to sign off volunteers as having received training for a specific role
-* Manager - Check volunteers arrive for shifts, provide aid
-* Admin - Create shifts/roles etc.
+* Volunteer Admin - Create shifts/roles etc.
 
-## Shift types
-Not the same as roles. Roles are e.g. "bar", "A/V" shifts are the specific combination of time, place & role.
-
-* Normal - A time period, at a location where a role to be filled.
-* Training - Special shift that the volunteer must complete before being allowed to do other shifts of that type (e.g. bar training)
```

Comment:
This was added mostly as a nice to have and was intended to allow e.g. the a/v team to have training sessions on their gear. It may be worth checking with that team how they'd like to do things

Reply:
Current thinking is that only bar is going to be tracking training through volunteer system (and they're a special case for how they're doing training) as they're the only ones who have in the past. 

But definitely open question on if that is because nobody else wants to handle training through the system or just that it didn't really work in the past so they couldn't. Need to ask if anyone else wants to track volunteer training here and how they'd want it to work.